charts/forseti-security/templates/server/database-secret.yaml

```diff
@@ -20,6 +20,6 @@ metadata:
     component: forseti-server
     chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
 type: Opaque
-data:
+stringData:
```

Comment:
@hiloboy0119 Thanks for this PR. Can you explain what the issue is and how to reproduce? Have you tried base64 encoding the username and password? This is how Terraform calls Helm and I have verified that works.